README.md

```diff
@@ -2,35 +2,58 @@
 Tileserver of basemap for applications.
 
 ## Tools
-- [Maptiler/tileserver-gl docker](https://hub.docker.com/r/maptiler/tileserver-gl)
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/)
+    - Native setup is possible for the tileserver and nginx. However, these instructions are for docker container setup.
+- [Maptiler/tileserver-gl-light docker](https://hub.docker.com/r/maptiler/tileserver-gl-light)
 - [OpenMapTiles Positron Style](https://github.com/openmaptiles/positron-gl-style/blob/master/style.json) & [Planning Positron Style](https://github.com/NYCPlanning/labs-gl-style/blob/master/data/style.json)
-    - OpenMapTiles version is used in application
-    - Planning version is kept for reference
 - [Nginx docker](https://hub.docker.com/_/nginx)
 - [Certbot CLI](https://eff-certbot.readthedocs.io/en/stable/using.html#certbot-command-line-options)
-- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/)
-    - Native setup is possible for the tileserver and nginx. However, these instructions are for docker container setup.
 
 ## Setup
 ### Configure the Mapbox vector tiles (mbtiles)
-The application is designed to serve [mapbox vector tiles](https://github.com/mapbox/vector-tile-spec). The tiles are not tracked in source because they can be hundreds or thousands of megabytes each. (Depending on the source of the files, they may also be subject to licenses which restrict their distribution.) The application is configured to look for a generic "basemap.mbtiles" file. For general users, the mbtiles should be sourced, loaded into the `tileserver` folder, and renamed to `basemap.mbtiles`. For DCP uses, these data are stored in a private Digital Ocean space. As of 05 July 2023, the application uses northeast data from 2017.
+The application is designed to serve [mapbox vector tiles](https://github.com/mapbox/vector-tile-spec). The tiles are not tracked in source because they can be hundreds or thousands of megabytes each. (Depending on the source of the files, they may also be subject to licenses which restrict their distribution.) The application is configured to look for a generic "metro-region.mbtiles" file.
 
-The `basemap.mbtiles` file should be created before running the rest of the application.
+#### Tile generation
+The `metro-region.mbtiles` file should be created before running the rest of the application.
+
+Download united states data into the top-level `data` folder.
+```bash
+wget -O data/us.osm.pbf https://download.geofabrik.de/north-america/us-latest.osm.pbf
```

Comment:
Are these instructions meant to be used for running the project locally? Asking because Macs don't have `wget`, at least not by default.

Reply:
Changed to `curl`.

But yes, we're running it locally on a `*nix` system. In future iterations, we may migrate to an Ubuntu-based runner.